hangman.js

```diff
@@ -1 +1,30 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("welcome to hangman");
+
+var game = {
+  word: "",
+  //guesses: [],
+  wordLetters: [],
+  maskedLetters: [],
+  getWord: function() {
+    this.word = prompt("Enter your secret word.");
+  },
+  splitWord: function() {
+    this.wordLetters = this.word.split("");
+    for (i = 0; i < this.word.length; i++) {
```

Comment:
In your for loop, I would make sure to include` var i = 0;` so i is not a global variable!